cmd/kubevirt-csi-driver/kubevirt-csi-driver.go

```diff
@@ -184,3 +184,31 @@ func parseLabels() map[string]string {
 
 	return infraClusterLabelsMap
 }
+
+// resolveNodeID resolves the infra cluster VM name and namespace from the node's providerID or annotations.
+// It returns the nodeID in the format "namespace/name" or an error if resolution fails.
+func resolveNodeID(nodeName, providerID string, annotations map[string]string) (string, error) {
```

Comment:
I think we can cut down even more, just exit (klog.Fatal I believe) if this returns an err. No point in starting the services (if the motivation is end user debugging, I doubt they need it)

Reply:
thanks. done